src/fetch_candidates.php

```diff
@@ -0,0 +1,70 @@
+<?php
+session_start();
```

Comment:
Remove this. `session-handler.php` is already included. Also, this is not a page but rather a script that works on the server side. Move this inside the `includes` folder.